vizro-core/docs/pages/user-guides/run.md

```diff
@@ -34,6 +34,27 @@ Dash is running on http://127.0.0.1:8050/
  * Debug mode: on
 ```
 
+??? info "Automatic reloading and debugging"
+
+    #### Reloading the dashboard
```

Comment:
```suggestion
```

Reply:
Having subsections in a callout adds them to the page navigation which feels a bit weird to me.
![image](https://github.com/mckinsey/vizro/assets/49395058/25b47033-425f-4942-a743-a072394dc5cd)

Probably we can turn this feature off somehow, but I actually think this callout doesn't need two different sections anyway so I'd suggest removing it.